### PR TITLE
ras survey

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

**Review by hamidnesar — APPROVED** [review]

**Review by puzikovam2 — APPROVED** [review]
reviewed and approved puzikovam2

**Review by sarwarahmed1 — APPROVED** [review]

**Review by georgeb79 — APPROVED** [review]

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Survey_Data_Verification_Steps.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_Survey_Data_Verification_Steps.java:1]: Please remove the extra space on lines 12, 19, 43, 44, 53, 82, 90, 92

**Review by matakevin — CHANGES_REQUESTED** [review]
remove the spaces and push code again, aside from that it looks good

**Review by matakevin — APPROVED** [review]
LGTM

**Pull request merged**